jib-core/src/main/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslator.java

```diff
@@ -216,19 +216,16 @@
       return ImmutableList.of();
     }
 
+    ImmutableList.Builder<AbsoluteUnixPath> listBuilder = ImmutableList.builder();
```

Comment:
tiny nit: Generally, would prefer a more descriptive name here like `volumeList`. Less descriptive names may be fine, but this variable spans 10 lines, so would help readers remember what the list contained by line 228.

Reply:
Thanks for the tips @coollog and the review! 

I'll try my max to practice more these conventions.